skytemple_randomizer/randomizer/util/util.py

```diff
@@ -121,6 +121,20 @@ def get_allowed_md_ids(conf: RandomizerConfig, with_plus_600=False, *, roster=Ro
         return fun.get_allowed_md_ids(ents, roster)
     return list(ents)
 
+def get_allowed_md_starter_ids(conf: RandomizerConfig, with_plus_600=False, *, roster=Roster.DUNGEON) -> List[u16]:
```

Comment:
This should default to Roster.STARTERS instead.

Reply:
Resolved by cce8957